# beaconGeometry.msg
Header header
bool canSeeBeacon
int32 positionX
int32 positionY
float64 angle
